BotCommands/LoLCommands.cs

```diff
@@ -20,7 +25,6 @@ public async Task HelpLol(CommandContext ctx)
         public async Task GetGameDataLoL(CommandContext ctx, string msg)
         {
             var gameData = LoLGameData.GameDataFill(msg);
-
             var embed = new DiscordEmbedBuilder
```

Comment:
Yoy can put message formatting code in a separate method for example in some class responsible only for formatting pretty responses.